locust/runners.py

```diff
@@ -678,12 +692,12 @@ def start(self, user_count: int, spawn_rate: float, **kwargs) -> None:
 
```

Comment:
Maybe remove the high spawn rate warning on line 680, it shouldnt be a problem any more, right?

Reply:
Yes, of course! Thanks. It's done in the other PR.